### PR TITLE
:sparkles: Add panel in biomes

### PR DESCRIPTION
**Pull request by Enoriz**
- Add panel sign in biomes
- Add elements in biomes prefab (less animals and panel sign)

image
image
image

### CODE REVIEW

File: Assets/Modules/SideEnvironment/Scripts/Panel.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace Aloha
+{
+    /// <summary>
+    /// Class for a panel sign
+    /// </summary>
+    public class Panel : SideEnvironment
+    {
+        /// <summary>
+        /// Initialize animal
```
Comment (DoBrcd) [inline Assets/Modules/SideEnvironment/Scripts/Panel.cs:12]:
```suggestion
        /// Initialize panel
```

File: Assets/Modules/SideEnvironment/Scripts/Panel.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        /// <example> Example(s):
+        /// <code>
+        ///     sideEnvInstR.Initialize();
```
Comment (DoBrcd) [inline Assets/Modules/SideEnvironment/Scripts/Panel.cs:15]:
```suggestion
        ///     panel.Initialize();
```

**Review by DoBrcd — APPROVED** [review]
Juste 2-3 modif de commentaire

**Comment by youenPlusquellec** [comment]
Trop de panneau, il faut les diminuer

**Pull request merged**